tests/datatypes/test_str.py

```diff
@@ -32,7 +32,7 @@ def test_isspace(self):
 
     def test_isalpha(self):
         self.assertCodeExecution("""
-            for s in ['Hello World', 'hello wORLd.', 'Hello world.', '',
+            for s in ['Hello World', 'hello wORLd.', 'Hello world.', '', 'hello1'
```

Comment:
@ashutoshsaboo did you mean to remove the comma after the hello1 string? Without it, 'hello1' and 'this' will be implicitly concatenated into 'hello1this' and I'm not sure that is what you want to test.